src/components/common/Map.js

```diff
@@ -146,20 +172,23 @@ class Map extends React.Component {
 
     // TARGETS RADIUS
     let circles = this.props.markers.map((venue) => {
+      const color = (this.props.targetSelected !== null &&
```

Comment:
The second condition implies the first condition.

If `this.props.targetSelected === venue)`, for sure `this.props.targetSelected !== null`

So, just the second condition will work